server/postProcessHandler/src/service/jobModel.ts

```diff
@@ -0,0 +1,26 @@
+export interface job {
```

Comment:
These interfaces don't have the type information that DynamoDB returns/uses (AttributeValue).